src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ThemeService.cs

```diff
@@ -159,6 +160,40 @@ public async Task<Either<ActionResult, Unit>> DeleteTheme(Guid themeId)
                 });
         }
 
+        public async Task<Either<ActionResult, Unit>> DeleteUITestThemes(CancellationToken cancellationToken = default)
+        {
+            return await _userService.CheckCanManageAllTaxonomy()
+                .OnSuccess(_ => _context.Themes
+                    .Where(theme => theme.Title.StartsWith("UI test theme") || theme.Title.StartsWith("Seed theme"))
+                    .Include(theme => theme.Topics))
+                .OnSuccessDo(CheckCanDeleteThemes)
+                .OnSuccessVoid(async themes =>
+                {
+                    foreach (var theme in themes)
+                    {
+                        var topicIds = theme.Topics.Select(topic => topic.Id).ToList();
+                        await topicIds.ToAsyncEnumerable().ForEachAwaitAsync(_topicService.DeleteTopic);
+
+                        _context.Themes.Remove(theme);
+                    }
+
+                    await _context.SaveChangesAsync();
+                    await _publishingService.TaxonomyChanged();
+                });
+        }
+
+        private async Task<Either<ActionResult, Unit>> CheckCanDeleteThemes(IEnumerable<Theme> themes)
+        {
```

Comment:
Mentioned in previously Slack - we'll want to do something similar to the current theme deletion endpoint where this endpoint can only be ran if the environment has been configured to do so.

See the `ThemeService` for how we're doing this.